.github/workflows/generate-cred.yml

```diff
@@ -0,0 +1,24 @@
+name: generate-cred
+
+on:
+  pull_request: []
```

Comment:
I know that `pull_request: {}` makes the action run on pull requests;
what does `pull_request: []` do? I’d assume that we don’t want this to
run on pull requests, in which case does it not suffice to just omit the
`pull_requests` key from the `on` object entirely?

Reply:
See above - this was just for testing.

Reply:
pull_request trigger is removed.